src/main/java/com/team/buddyya/common/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,19 @@
+package com.team.buddyya.common.exception;
+
+import com.team.buddyya.certification.exception.PhoneAuthenticationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(PhoneAuthenticationException.class)
+    public ResponseEntity<ErrorResponse> handlePhoneAuthenticationException(PhoneAuthenticationException ex) {
+        ErrorCode errorCode = ex.getErrorCode();
+        HttpStatus status = errorCode == ErrorCode.SMS_SEND_FAILED ? HttpStatus.INTERNAL_SERVER_ERROR : HttpStatus.BAD_REQUEST;
```

Comment:
이 부분에서 가독성을 위해 괄호 정도 넣어도 될 것 같아요~

```suggestion
        HttpStatus status = (errorCode == ErrorCode.SMS_SEND_FAILED) ? HttpStatus.INTERNAL_SERVER_ERROR : HttpStatus.BAD_REQUEST;
```